app/channels/timelines_channel.rb

```diff
@@ -3,6 +3,8 @@
 class TimelinesChannel < ApplicationCable::Channel
   before_subscribe :set_host_for_disk_storage
 
+  SEND_TIMELINES_LIMIT_NUM = 20
```

Comment:
```suggestion
  TIMELINES_LIMIT = 20
```

とか、`TimelinesChannel::PER_PAGE` とかでもいいかも。

Reply:
@komagata 
定数名を`TIMELINES_LIMIT`に変更しました。
レビューよろしくお願いいたします。